submissions/usides/html-css-popup/css/main.css

```diff
@@ -0,0 +1,376 @@
+@import url("https://fonts.googleapis.com/css2?family=Rajdhani&family=Red+Hat+Display&family=Russo+One&display=swap");
+button,
+a,
+input,
+li {
+  outline: none;
+  border: transparent;
+  background-color: rgba(255, 255, 255, 0);
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+body {
+  background-image: url(../img/cat_bg.jpg);
+  background-position: 22% 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+  min-height: 100vh;
+}
+
+nav ul {
+  margin: 0;
+  list-style-type: none;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  padding: 0.5rem;
+  padding-right: 1.5rem;
+}
+
+nav ul li > a {
+  margin: 0.3rem;
+}
+
+nav ul .direct-link {
+  font-family: 'Rajdhani', sans-serif;
+  font-size: 1.2rem;
+  text-decoration: none;
+  font-weight: bold;
+  color: #fff;
+  padding: 0 0.2rem;
+  min-height: 1.7rem;
+  -webkit-transition: all 0.2s;
+  transition: all 0.2s;
+}
+
+nav ul .direct-link:focus {
+  border-bottom: 3px solid #423e3e;
+}
+
+nav ul .menu {
+  position: relative;
+  margin: 0 0.3rem;
+}
+
+nav ul .menu input#btn-app {
+  position: absolute;
+  opacity: 0;
+}
+
+nav ul .menu input#btn-app:checked ~ div.hidden {
+  visibility: visible;
+}
+
+nav ul .menu input#btn-app:focus ~ .menu-btn-app {
+  background-color: #423e3e;
+}
+
+nav ul .menu .menu-btn-app {
+  display: block;
+  max-width: 2.5rem;
+  padding: 0.2rem 0;
+  padding: 0.4rem;
+  border-radius: 0.2rem;
+  outline: none;
+  cursor: pointer;
+  -webkit-transition: all 0.2s;
+  transition: all 0.2s;
+}
+
+nav ul .menu .triangle {
+  visibility: hidden;
+  position: absolute;
+  top: 96%;
+  left: 43%;
+  margin-left: -5px;
+  border-width: 8px;
+  border-style: solid;
+  border-color: transparent transparent #0000004d transparent;
+}
+
+nav ul .menu .menu-popup {
```

Comment:
Avoid creating too specific selectors. In a big project, it's hard to support or override by another selector.
Use `class` for adding styles to element.
[Read more about selector specificity.](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
I used scss preprocessor and nesting feature. How can I fix this? Should I remove some nesting levels?

Reply:
1. PR must contain source files(.scss in your case) not compiled. For this PR leave current files, for future PR's add source files instead of ready compiled files.
2. Yes, remove some nested levels, As bigger nesting as much more time need to browser for rendering your code.